addons/fleet/models/fleet_vehicle.py

```diff
@@ -0,0 +1,553 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from collections import defaultdict
+import datetime
+from dateutil.relativedelta import relativedelta
+
+from openerp import api, fields, models, _
+from openerp.exceptions import UserError
+
+
+class FleetVehicle(models.Model):
+
+    _name = 'fleet.vehicle'
+    _description = 'Information on a vehicle'
+    _order = 'license_plate asc'
+    _inherit = 'mail.thread'
+
+    @api.model
+    def default_stage(self):
+        try:
+            stage = self.env.ref('fleet.vehicle_stage_active')
+        except ValueError:
+            stage = False
+        return stage
```

Comment:
remove try..expect and replace it with,
return self.env.ref('fleet.vehicle_stage_active', raise_if_not_found=False)